wasm/wasm_source/Makefile

```
@@ -0,0 +1,65 @@
+cargo := $(env) cargo
+rustup := $(env) rustup
+# Nightly build is currently used for rustfmt and clippy.
+nightly := $(shell cat ../../rust-nightly-version)
+
+# All the wasms that can be built from this source, switched via Cargo features
+# Wasms can be added via the Cargo.toml `[features]` list.
+wasms := tx_from_intent
+wasms += tx_transfer
+wasms += tx_update_vp
+wasms += vp_token
+wasms += vp_user
+wasms += mm_filter_token_exch
+wasms += mm_token_exch
+
+# Build all wasms
+all: $(wasms)
+
+# `cargo check` all wasms
+check: 
```

just a trivial stuff
```suggestion
check:
```